### PR TITLE
Fix Undo Redo Stack

### PR DESCRIPTION
**Pull request by markwindsorr**
Addresses issue where while using a notebook, you are unable to undo or redo a cut or paste option. Change should be tested as I am unable to run the react package locally.

### CODE REVIEW

File: packages/react/src/components/notebook/NotebookCommands.ts [hunk 1]
```diff
@@ -274,13 +275,38 @@ export const NotebookCommands = (
     execute: () => NotebookActions.splitCell(notebookPanel.content),
   });
   commandRegistry.addCommand(cmdIds.undo, {
```
Comment (markwindsorr) [inline packages/react/src/components/notebook/NotebookCommands.ts:277]: not sure whats going on with the indentation here

Comment (markwindsorr) [inline packages/react/src/components/notebook/NotebookCommands.ts:277]: not sure whats going on with the indentation here

**Comment by echarles** [comment]
@markwindsorr Thx a lot. Changes look good and CI build is fine. about the indentation. just replied for the indentation.

File: packages/react/src/components/notebook/NotebookCommands.ts (hunk 1 above)
Comment (echarles) [reply packages/react/src/components/notebook/NotebookCommands.ts:277]: probably related to the change below in this file. I you don't know, that is fine, I can merge as such and look after.

Comment (markwindsorr) [reply packages/react/src/components/notebook/NotebookCommands.ts:277]: im using a linter with prettier, but its looking fine on my end but showing up with weird indentation on the pr.

Comment (echarles) [reply packages/react/src/components/notebook/NotebookCommands.ts:277]: prolly a github display issue. will resolve this.

**Review by echarles — APPROVED** [review]
LGTM Thx @markwindsorr

**Pull request merged**